packages/IMAPClient-Tests.package/ICMockCommandStorage.class/class/new.st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+new 
+	
+	^ super new copy removeAll
```

Comment:
if you find a better solution for removing the "Singleton nature" of ICMockCommandStorage feel free to replace this line or make suggestion!

Reply:
No, I think this is okay. @sm1lla, please merge! ^^